wordpress-mobile/WordPress-iOS
# Web authentication improvements

Pre-authenticate reader posts opened in a web view.
Add a `rememberme` parameter to log in, so the cookies are persisted.
Improve cookie URL matching.

Second part of #8207.
Fixes #8191 

To test:

1. Go to the reader and find a post
2. Tap on the Visit link to see the post full screen.
3. There should be a logged in bar.
4. Find the like button and tap on it. It should work without presenting a login screen.
5. Dismiss the web view.
6. Visit link again, paying attention to the view's title. It should load the post directly, and not show the `wordpress.com` URL first that indicates the login redirect.
